### DISCUSSION TITLE
inputTag能否把tag的close和input的readonly分离

### DISCUSSION THREAD

**Discussion by DaYang816** [element-plus/element-plus#19425] [Ideas]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `input-tag`

## Description

请教一下，inputTag能否把tag的close和input的readonly分离,我希望input只读的时候是无法使用输入input去创建tag，可以使用别的逻辑去控制input中tag的值，并且tag的关闭按钮显隐开放使用字段控制。
我希望可以控制tag，并且关闭input输入创建tag，tag可以点击x进行删除，
谢谢。

### Additional comments

_(empty)_

**Comment by tolking** [comment]
如果 tag 可以删除不是就改变 modelValue 值了吗。和 readonly 的功能冲突

**Reply by DaYang816** [reply]
好的，谢谢回复。
不过确实会出现不希望输入框的去重新新建出tag的场景，同时也要有修改的操作，添加tag是从自己写的逻辑出来的。
只是希望有个开关去自己控制这块，tag的close自己也能控制。
目前看源码是没有开放出来的，是根据readonly一并控制。组件写的很不错。
可能是我的想法有点既要又要的意思，自己的设想是能自主控制，可控制是否焦点输入可自己控制close，新增可通过自己的逻辑，例如弹窗有个选项去选择后新增，tag可删除，害怕输入框然后回车会去写入不是选择的值。
